app/src/main/java/de/westnordost/streetcomplete/quests/barrier_bicycle_barrier_installation/AddBicycleBarrierInstallation.kt

```diff
@@ -0,0 +1,37 @@
+package de.westnordost.streetcomplete.quests.barrier_bicycle_barrier_installation
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.BICYCLIST
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.BLIND
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.WHEELCHAIR
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.LIFESAVER
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddBicycleBarrierInstallation : OsmFilterQuestType<BicycleBarrierInstallationAnswer>() {
+
+    override val elementFilter = "nodes, ways with barrier = cycle_barrier and cycle_barrier and cycle_barrier != tilted and !cycle_barrier:installation"
```

Comment:
Please format this into a multiline string for better readability

Reply:
done